it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersCorporateBodyCompanyRegistrationNumberControllerISpec.scala

```diff
@@ -0,0 +1,95 @@
+package uk.gov.hmrc.fhregistrationfrontend.controllers
+
+import org.jsoup.Jsoup
+import play.api.libs.ws.DefaultWSCookie
+import play.api.test.WsTestClient
+import uk.gov.hmrc.fhregistrationfrontend.testsupport.{Specifications, TestConfiguration}
+
+class BusinessPartnersCorporateBodyCompanyRegistrationNumberControllerISpec
+  extends Specifications with TestConfiguration {
+
+  "GET /form/business-partners/company-registration-number" should {
+    "render the corporate-body-company-registration-number page" in {
+      given
+        .commonPrecondition
+
+      WsTestClient.withClient { client =>
+        val result = client.url(s"$baseUrl/form/business-partners/company-registration-number")
+          .addCookies(DefaultWSCookie("mdtp", authAndSessionCookie))
+          .get()
+
+        whenReady(result) { res =>
+          res.status mustBe 200
+          val page = Jsoup.parse(res.body)
+          page.title() must include("What is the company’s company registration number?")
+          page.getElementsByTag("h1").text() must include("What is test corporateBody’s company registration number?")
+          page.getElementsByTag("p").text() must include("You can search Companies House for a company registration number (opens in new tab)")
+        }
+      }
+    }
+  }
+
```

Comment:
Is is worth adding an IT test for 6 numbers and two letters also ? Aka 12345AB ?